### DISCUSSION TITLE
MiXCR Built in Library

### DISCUSSION THREAD

**Discussion by bshim181** [milaboratory/mixcr#1575] [General]
Hello, 

I am currently aware that I can get list of V-gene/V-transcript sequences from MiXCR Gene Library. I was wondering if there is a way to access the reference points for each of the V gene features in the form of csv or tsv files. I would love to know the breakpoints for each of these gene features so I can possibly look at these features individually.

**Comment by mizraelson** [comment]
Hi, there is no way to export this in a table format from the library. The CDRs and FRs are only shown in the interface. If you have a dataset, and use mixcr to analyze it, you can export a germline sequence for different regions in the output using e.g.:
`-nFeature CDR2 germline` parameter for exportClones.
